doc/howtos/rdtraining/firstui.rst

```diff
@@ -0,0 +1,274 @@
+.. _howto/rdtraining/firstui:
+
+=============================
+Finally, Some UI To Play With
+=============================
+
+After the creation of a new :ref:`model <howto/rdtraining/basicmodel>` and its corresponding
+:ref:`access rights <howto/rdtraining/securityintro>`, it is now time to be able to interact in
+the user interface.
+
+At the end of this chapter, a couple of menus will be created in order to access a default list
+and form view.
+
+Data Files (XML)
+================
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/data`.
+
+In the :ref:`howto/rdtraining/securityintro` chapter, we added data through a CSV file. The CSV
+format is convenient when the data to load has a simple format. If the format is more complex
+(*e.g.* load the structure of a view or an email template), we use the XML format. For example,
+this
+`help field <https://github.com/odoo/odoo/blob/09c59012bf80d2ccbafe21c39e604d6cfda72924/addons/crm/views/crm_lost_reason_views.xml#L61-L69>`__
+contains HTML tags. While it would be possible to load such data through a CSV, it is more
+convenient to use an XML file.
+
+The XML files must be added in the same folders than the CSV files, and defined similarly in the
+``__manifest__.py``. The content of the data files is also loaded when a module is installed or
+updated, sequencially. Therefore, all remarks made for the CSV files holds true for the XML files.
+
+In this chapter, we will load our first action and menus though an XML file. Actions and menus are
+regular records in database.
+
+Actions
+=======
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/actions`.
+
+.. note::
+
+    **Goal**: at the end of this section, an action should be loaded in the system. We won't see
+    anything yet in the UI, but the file should be loaded in the log:
+
+    .. code-block:: text
+
+        INFO rd-demo odoo.modules.loading: loading estate/data/estate_property_views.xml
+
+Actions can be triggered in three ways:
+
+1. by clicking on menu items (linked to specific actions)
+2. by clicking on buttons in views (if these are connected to actions)
+3. as contextual actions on object
+
+We will only cover the first case in this chapter. The second case will be covered in an
+:ref:`upcoming chapter <howto/rdtraining/actions>` while the last one is the topic of an appendix.
+In our Real Estate example, we would like to link a menu to the ``estate.property`` model, so we
+are able to create a new record. The action can be seen as the link between the menu and
+the model.
+
+A minimal action for our ``test.model`` would look like:
+
+.. code-block:: xml
+
+    <record id="test_model_action" model="ir.actions.act_window">
+        <field name="name">Test action</field>
+        <field name="res_model">test.model</field>
+        <field name="view_mode">tree,form</field>
+    </record>
```

Comment:
```suggestion
   <odoo>
      <record id="test_model_action" model="ir.actions.act_window">
          <field name="name">Test action</field>
          <field name="res_model">test.model</field>
          <field name="view_mode">tree,form</field>
      </record>
   </odoo>
```

Or explain before that all the xml files must start with <odoo> and finish with </odoo>

Reply:
Added more explanation before the exercise